sdks/python/apache_beam/examples/snippets/snippets.py

```diff
@@ -845,6 +845,30 @@ def filter_words(x):
   p.run().wait_until_finish()
 
 
+def model_textio_compressed(renames):
+  """Using a Read and Write transform to read/write text files."""
+  def filter_words(x):
+    import re
+    return re.findall(r'[A-Za-z\']+', x)
+
+  import apache_beam as beam
+  from apache_beam.utils.pipeline_options import PipelineOptions
+
+  # [START model_textio_write_compressed]
+  p = beam.Pipeline(options=PipelineOptions())
```

Comment:
There's no need for this to be part of the snippet. That will let you use TestPipeline.